metaflow/plugins/kfp/tests/Dockerfile

```diff
@@ -1,8 +1,13 @@
-FROM analytics-docker.artifactory.zgtools.net/artificial-intelligence/ai-platform/aip-py39-cpu:4.0.1380
+FROM analytics-docker.artifactory.zgtools.net/artificial-intelligence/ai-platform/aip-py39-cpu-cicd:4.0.1380
```

Comment:
why use the cicd image?

Reply:
The CICD image comes with `kfp` (more specifically `aip-kfp-sdk`), which the regular image doesn't. This [error](https://gitlab.zgtools.net/analytics/artificial-intelligence/ai-platform/aip-workflow/aip-metaflow-integration-testing/-/jobs/8273494) came up when using the regular image. I believe Alex moved `aip-kfp-sdk` to the CICD images as part of the upgrade.

Reply:
Instead of linking to the error and calling it an "error", could you instead please share a description of it?

Reply:
Error: `ModuleNotFoundError: No module named 'kfp'`

Reply:
Depending on a cicd image is finicky... I'd rather us pip install `kfp` or `aip-kfp-sdk`